src/flag_gems/ops/randn.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from flag_gems.utils.random_utils import philox_cuda_seed_offset, uint_to_uniform_float
+from flag_gems.utils.shape_utils import volume
+
+try:
+    pair_uniform_to_normal = tl.pair_uniform_to_normal
+except AttributeError:
+
+    @triton.jit
+    def pair_uniform_to_normal(u1, u2):
+        """Box-Muller transform"""
+        u1 = triton.standard.maximum(1.0e-7, u1)
```

Comment:
It should be `tl.maximum` or `tl.language.maximum`.

Reply:
thanks～